packages/schematics/angular/package.json

```diff
@@ -2,6 +2,7 @@
   "name": "@schematics/angular",
   "version": "0.0.0",
   "description": "Schematics specific to Angular",
+  "homepage": "https://github.com/angular/angular-cli",
```

Comment:
`homepage` field is needed so that relative links are generated correctly on NPM